2022-10-09 15:46:07
Traceback (most recent call last):
  File "\\?\C:\Users\will\AppData\Roaming\Sublime Text\Packages\SFTP\sftp\threads.py", line 22, in run_with_except_hook
  File "\\?\C:\Users\will\AppData\Roaming\Sublime Text\Packages\SFTP\sftp\commands.py", line 2281, in run
  File "./python3.8/shutil.py", line 709, in rmtree
  File "./python3.8/shutil.py", line 707, in rmtree
FileNotFoundError: [Errno 2] No such file or directory: '/home/alan/CEIBA/ceiba-carwler/vendor/phpunit/'
